.github/workflows/validate-snapshots.yml

```diff
@@ -0,0 +1,67 @@
+name: Validate snapshots
+on:
+  workflow_dispatch:
+  schedule: 
+    - cron: '30 9 * * 1-5'
```

Comment:
Could run slightly after 9.30. Publishing isn't guaranteed to happen exactly at 9.30 precisely because the Publisher function app has to initialise and do some processing. I suppose this action takes some time to do it's job too though so maybe it's ok?

Because the published data can change at any time during the day not just at 9.30, depending on what the reviewers think to the possibility of being hit by PR's at different times of the day, this could run say 2 or 3 times during the day?

I think that's a good idea also because this is ultimately a testing tool, and bugs with the data rendered by the site could present themselves at any time during the day, and not just when releases are published/amended, and not limited to weekdays. For that reason I think it could also be beneficial to run at weekends.

Reply:
Unfortunately due to the caching of dependencies, this workflow kicks off pretty quick. I'll schedule it to run at 9:35AM instead. Yep, sounds good to run it at other times during the day when releases are getting published. Something such as 9:30AM, 12PM, 4PM mon-fri?

Reply:
Cool, the proposed schedule sounds good to me 🙂 .